docs/argument-constraints.md

```diff
@@ -111,6 +111,20 @@ events in the context of an `A.CallTo`.
 So, tempting as it might be to save one of the constraints away in a
 handy variable, don't do it.
 
+## Using correct grammar
+
+FakeItEasy's API attempts to imitate the English language, so that call
+configurations and assertions read naturally. In that spirit, it's
+possible to use `An<T>` instead of `A<T>` for types whose name starts
+with a vowel. For instance:

```

Comment:
Arguably "starts with a vowel _sound_."  `An<Apple>` is a great use of this. `An<OuijaBoard>` less so.